### DISCUSSION TITLE
Repeat Option for Timer

### DISCUSSION THREAD

**Discussion by clay53** [InfiniTimeOrg/InfiniTime#1626] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

My cross country/track team uses watches that vibrate automatically every like 17-30s (depending on your pace group) to keep you on pace per 100m. This would be very useful for me and likely other runners.

### Description

Basically, set the timer to what you want the repeat interval to be, then, with "repeat mode" enabled, whenever the timer hits 0, the watch immediately vibrates and starts the timer again at the same number. Another button next to start/stop button in the normal timer app to toggle "repeat mode" would be an easy way to implement this. However, for my use case, the touch button to start/stop the button is very inconvenient. If a "tap" of the hardware button was able to start/stop it that would be very nice (exiting would then just be a hold) but that should probably be part of a separate issue.

**Comment by rimugu** [comment]
A vote for this request!

**Comment by GeckoEidechse** [comment]
Came across this discussion while looking for existing feature requests for this.

To expand on this. Outside of running, in general for any kind of exercise that consists out of repeating time intervals (stretching different muscle groups for X seconds before/after workout comes to mind), such a feature would a blessing.
Interrupting your stretch every time the timer goes off and setting it back to e.g. 30 seconds is quite a nuisance to the point where it does not make sense to use the PineTime for this.

If there's anything to get this feature pushed forward, I'd be happy to help <3

**Comment by NordMike** [comment]
This could also be achieved by the metronome app if it allowed lower BPM.
